plugins/techdocs-react/src/context.tsx

```diff
@@ -96,6 +102,7 @@ export type TechDocsReaderPageProviderRenderFunction = (
  * @public
  */
 export type TechDocsReaderPageProviderProps = {
+  path?: string;
```

Comment:
Maybe this needs to be required? And if it is, the changes is probably breaking?

Reply:
I added fallback for path and entityId which are derived from URL, what do you think?